### DISCUSSION TITLE
Firecrawl Documentation for Flowise

### DISCUSSION THREAD

**Discussion by taowang1993** [FlowiseAI/Flowise#3028] [General]
Here is Dify's docs on integrating Firecrawl: 
https://docs.dify.ai/guides/knowledge-base/sync-from-website#id-1.-configure-firecrawl-api-credentials

I am wondering if it is works the sames on Flowise. 

The Flowise documentation lacks this part.

Thank you.

**Comment by HenryHengZJ** [comment]
Yep! You just need to get an API key from Firecrawl and use it within Flowise

**Reply by taowang1993** [reply]
> Yep! You just need to get an API key from Firecrawl and use it within Flowise

Does Flowise support "excluding heading, navs, footers, etc"? 

I only want to scape the main content of a documentation website, not the left manu bar, the top navbar and the footer.

This way, the knowledge base would be much more cleaner.

![image](https://github.com/user-attachments/assets/0535d73b-d151-4298-9e74-b93d9cf08fed)
